src/common/BaseAnnotator.ts

```diff
@@ -71,6 +72,7 @@ export default class BaseAnnotator extends EventEmitter {
             annotations: {
                 activeId: fileOptionsValue?.annotations?.activeId ?? null,
             },
+            common: { mode: initialMode ?? store.Mode.NONE },
```

Comment:
Do we need this ?? or can we let the undefined value be handled by the reducer’s default value?